okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/Okhttp39HostnameVerifier.java

```diff
@@ -79,7 +79,7 @@ public boolean verify(String host, SSLSession session) {
     }
 
     public boolean verify(String host, X509Certificate certificate) {
-        return verifyAsIpAddress(host) ? verifyIpAddress(host, certificate) : verifyHostname(host, certificate);
+        return canParseAsIpAddress(host) ? verifyIpAddress(host, certificate) : verifyHostname(host, certificate);
```

Comment:
[Weird rename?](https://github.com/square/okhttp/commit/42a41f3857fb8a7f93fc2e1f5a8b4c534329fbff#diff-1d7cffa64af32fd577609a790c990e9ae9728cca3b1bbfd5175adf532a320567R235)

Reply:
nit: can we remove the static import and instead `import okhttp3.internal.Util;`

```suggestion
        return Util.canParseAsIpAddress(host) ? verifyIpAddress(host, certificate) : verifyHostname(host, certificate);
```